docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -103,22 +103,22 @@ Version 2.0.30
 
 ### Role
 
-* graph: support required access configuration and native client
+* `az ad app create`: support required access configuration and native client
```

Comment:
Still not sure what this description means or if it is a user-facing change.

Reply:
This summary may help - https://github.com/Azure/azure-cli/commit/4c92d1fb317db3e358a06f53ca6f5714a8f4e27d#diff-bde36151e74187cc6b61e16884149087R262